src/txn/txn_ckpt.c

```diff
@@ -1339,6 +1334,8 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
         } else
             conn->txn_global.last_ckpt_timestamp = WT_TS_NONE;
     }
+    __checkpoint_stats(session);
+    WT_STAT_CONN_INCR(session, checkpoints);
```

Comment:
This stat is increased in __session_checkpoint, which means it only tracks the number of checkpoints initiated by the api. It doesn't include the internal initiated checkpoints. If we increase it here, we should remove that in __session_checkpoint. In addition, we should move this to the err label, otherwise this only includes the successful checkpoints.